src/preload.py

```diff
@@ -538,16 +548,8 @@ def start_docker_daemon(storage_driver, docker_dir):
         if running_dockerd.process.exit_code is not None:
             # There is no reason for dockerd to exit with a 0 status now.
             assert running_dockerd.process.exit_code != 0
-            try:
-                # This will raise an sh.ErrorReturnCode_X exception.
-                running_dockerd.wait()
-            except ErrorReturnCode as e:
-                log.info(
-                    "An error has occurred executing 'dockerd':\n{}".format(
-                        e.stderr.decode("utf8"),
-                    ),
-                )
-                raise e
+            # This will raise an sh.ErrorReturnCode_X exception.
+            running_dockerd.wait()
```

Comment:
This change reverts to how it was before I changed it in the previous PR: https://github.com/balena-io-modules/balena-preload/pull/220/files#diff-5d1069846576354d1dab375d260d34c1L493

There was nothing wrong the the code, but the `log.info()` proved redundant after this PR introduced the `{"_truncate_exc": False}` to `sh`. (The `log.info()` was a workaround to show the full error output when `DEBUG=1` was set by the CLI.)